src/encoded/static/components/static-pages/components/benchmarking/BenchmarkingDataMap.js

```diff
@@ -181,130 +181,74 @@ export const BenchmarkingDataMap = {
         tabMapArray: [
             {
-                eventKey: '#donor-1',
-                title: 'Donor 1',
-                // searchHref: '/search/?type=File&status=released&status=restricted&status=public',
+                eventKey: '#liver',
+                title: 'Liver',
+                searchHref:
+                    '/search/?type=File&status=released&status=restricted&status=public&file_sets.libraries.analyte.samples.sample_sources.code=ST001-1A',
             },
             {
-                eventKey: '#donor-2',
-                title: 'Donor 2',
-                // searchHref: '/search/?type=File&status=released&status=restricted&status=public',
+                eventKey: '#lung',
+                title: 'Lung',
+                searchHref:
+                    '/search/?type=File&status=released&status=restricted&status=public&file_sets.libraries.analyte.samples.sample_sources.code=ST001-1D',
             },
         ],
     },
-    Liver: {
-        navBarTitle: 'Liver',
-        title: 'Liver Primary Tissue Data',
+    Donor2: {
+        navBarTitle: 'Donor ST002',
+        title: 'Donor ST002 : Benchmarking Tissue Data',
         description: primaryTissuePageDescription,
         type: 'Primary Tissue Data',
-        path: '/data/benchmarking/liver',
+        path: '/data/benchmarking/donor-st002',
         tabMapArray: [
             {
-                eventKey: '#donor-1',
-                title: 'Donor 1',
-                // searchHref: '/search/?type=File&status=released&status=restricted&status=public',
+                eventKey: '#colon',
+                title: 'Colon',
+                searchHref:
+                    '/search/?type=File&status=released&status=restricted&status=public&file_sets.libraries.analyte.samples.sample_sources.code=ST002-1G',
             },
             {
-                eventKey: '#donor-2',
-                title: 'Donor 2',
-                // searchHref: '/search/?type=File&status=released&status=restricted&status=public',
+                eventKey: '#lung',
+                title: 'Lung',
+                searchHref:
+                    '/search/?type=File&status=released&status=restricted&status=public&file_sets.libraries.analyte.samples.sample_sources.code=ST001-1D',
             },
         ],
     },
-    Colon: {
-        navBarTitle: 'Colon',
-        title: 'Colon Primary Tissue Data',
+    Donor3: {
+        navBarTitle: 'Donor ST003',
+        title: 'Donor ST003 : Benchmarking Tissue Data',
         description: primaryTissuePageDescription,
         type: 'Primary Tissue Data',
-        path: '/data/benchmarking/colon',
+        path: '/data/benchmarking/donor-st003',
         tabMapArray: [
             {
-                eventKey: '#donor-1',
-                title: 'Donor 1',
-                // searchHref: '/search/?type=File&status=released&status=restricted&status=public',
+                eventKey: '#brain',
+                title: 'Brain',
```

Comment:
Any reason why we're not providing the searches for the brain tissues here and below? Relevant codes are `ST003-1Q` and `ST004-1Q`, resp.

Reply:
Since we won't have those right away, Elizabeth wants to display "coming soon". And then we'll update later with the real URLs.